README.md

```diff
@@ -259,7 +260,7 @@ sudo make package
 
 ## Windows 10 (x64)
 
-Currently only the documentation can't be built on Windows.
+Currently only the documentation cannot be built on Windows.
```

Comment:
I didn't get the meaning of the comment really.

Reply:
It means everything but the documentation builds on Windows.  Many things in the documentation could be more clear for sure.